cyqldog/db.go

```diff
@@ -94,7 +93,7 @@ func buildRecord(row []interface{}, cols []string) (Record, error) {
 	for i, c := range row {
 		s, err := convertToString(c)
 		if err != nil {
-			return record, errors.Wrapf(err, "failt to convertToString: col = %s, type = %T(%v)", cols[i], c, c)
+			return record, xerrors.Errorf("faied to convertToString: col = %s, type = %T(%v): %w", cols[i], c, c, err)
```

Comment:
Both the original and the fixed code are typos :)

Reply:
ohh...
Fixed typo in cc4fc81